tests/test_buffers.cpp

```diff
@@ -7,12 +7,40 @@
     BSD-style license that can be found in the LICENSE file.
 */
 
+#include <pybind11/complex.h>
 #include <pybind11/stl.h>
 
 #include "constructor_stats.h"
 #include "pybind11_tests.h"
 
 TEST_SUBMODULE(buffers, m) {
+
+#define PYBIND11_LOCAL_DEF(...)                                                                   \
```

Comment:
Not sure if it's worth changing, but I tend to not like macros that return values.

I would have written this as:

```
std::map<std::string, std::string> values;

#define ASSIGN_HELPER(...)
    values[#__VA_ARGS__] = return py::format_descriptor<__VA_ARGS__>::format();

ASSIGN_HELPER(bool);
ASSIGN_HELPER(PyObject*);

return values[cpp_name];
```

Reply:
Changed.